tests/alfacase/test_case_description.py

```diff
@@ -1374,3 +1374,71 @@ def test_case_description_duplicate_names_between_elements(default_well):
 
     with pytest.raises(InvalidReferenceError, match=re.escape(expected_msg)):
         case.ensure_unique_names()
+
+
+def test_check_fluid_references(default_well: case_description.WallDescription) -> None:
+    """
+    Test _check_fluid_references isn't yielding errors for a correct CaseDescription.
+    """
+
+    known_pvt_properties = set(
+        property
+        for property in vars(case_description.PvtModelsDescription).keys()
+        if not property.startswith("__") and not property.endswith("__")
+    )
```

Comment:
I think you could just do:

```suggestion
    class_ = case_description.PvtModelsDescription
    known_pvt_properties = set(attr.fields_dict(class_).keys())  # The `.keys()` is not really required.
```